net/disk_cache/disk_cache_backend.cc

```diff
@@ -1,391 +0,0 @@
-// Copyright (c) 2012 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-#ifndef USE_HACKY_COBALT_CHANGES
-#error Many definitions in this file should instead be defined in disk_cache.cc
```

Comment:
Sorry if this error message was description--this is saying that many things defined here, i.e. `BackendResult::BackendResult`, are just copied from `disk_cache,cc` and I didn't know if this was the right place to put them (I don't like the fact that there's so much repeated code). But many of the functions implemented in this file have a different implementation than in `disk_cache.cc`, so we can't just remove it entirely. I still don't know the clean solution here, but this is more of a cleanup issue than a functional issue, so it's not crucial to fix

Reply:
> But many of the functions implemented in this file have a different implementation than in disk_cache.cc

Are we using those differently implemented functions though?

Reply:
Yes we are for caching, @aee-google knows more about it though